src/Component/EdisMax.php

```diff
@@ -200,4 +200,31 @@ public function getUserFields()
     {
         return $this->getOption('userfields');
     }
+
+    /**
+     * Set SplitOnWhitespace option.
+     *
+     * Specifies whether the query parser splits the query text on whitespace before it's sent to be analyzed.
+     *
+     * The default is to split on whitespace, equivalent to &sow=true.
+     * The sow parameter was introduced in Solr 6.5.
+     *
+     * @param bool $splitOnWhitespace
+     *
+     * @return self Provides fluent interface
+     */
+    public function setSplitOnWhitespace($splitOnWhitespace)
+    {
+        return $this->setOption('splitonwhitespace', $splitOnWhitespace);
+    }
+
+    /**
+     * Get SplitOnWhitespace option.
+     *
+     * @return bool|null
+     */
+    public function getSplitOnWhitespace()
+    {
+        return $this->getOption('splitonwhitespace');
+    }
```

Comment:
According to https://lucidworks.com/2017/04/18/multi-word-synonyms-solr-adds-query-time-support/ the `sow`parameter is available for the `edismax` and the `lucene` query parsers.

Depending on your solrconf.xml, you use the lucene query parser if you don't add the EdisMax component to Solarium\QueryType\Select\Query

Therefore I suggest to move this functionality to Solarium\QueryType\Select\Query itself. I assume that the sow parameter simply doesn't hurt if someone sets it in combination with dismax for example.